### PR TITLE
kh360-react-443-progress-screen-mobile-view

### PR DESCRIPTION
**Pull request by chadiegilnerubia**
Ticket ID:
[View Evaluation Administration Progress Screen - adjust to make it mobile responsive#443](https://github.com/nerubia/kh360-react/issues/443)

### CODE REVIEW

File: src/features/admin/evaluation-administrations/[id]/progress/evaluation-progress-list.tsx [hunk 1]
```diff
@@ -44,6 +45,8 @@ export const EvaluationProgressList = () => {
   const [showEmailLogDialog, setShowEmailLogDialog] = useState<boolean>(false)
   const [selectedEvaluationId, setSelectedEvaluationId] = useState<number | null>(null)
 
+  // const NAME_LENGTH = 10
```
Comment (emmaacha) [inline src/features/admin/evaluation-administrations/[id]/progress/evaluation-progress-list.tsx:48]: I think this should be removed?

**Review by emmaacha — CHANGES_REQUESTED** [review]

File: src/features/admin/evaluation-administrations/[id]/progress/evaluation-progress-list.tsx (hunk 1 above)
Comment (chadiegilnerubia) [reply src/features/admin/evaluation-administrations/[id]/progress/evaluation-progress-list.tsx:48]: Yeapp2x I forgot to remove the constant my bad. :3

**Review by emmaacha — APPROVED** [review]
Cool!

File: __tests__/pages/admin/evaluation-administrations/[id]/progress/evaluation-progress.spec.ts [hunk 2]
```diff
@@ -218,8 +218,18 @@ test.describe("Admin - Evaluation Progress", () => {
       await expect(page.getByRole("heading", { name: "Evaluation Progress" })).toBeVisible()
       await expect(page.getByText("Evaluation 1")).toBeVisible()
       await expect(page.getByText("Pending")).toBeVisible()
-      await expect(page.getByText("Evaluation Period: 2023-01-01 to 2023-12-31")).toBeVisible()
-      await expect(page.getByText("Evaluation Schedule: 2024-01-01 to 2024-01-03")).toBeVisible()
+      if (!isMobile) {
```
Comment (jatnerubia) [inline __tests__/pages/admin/evaluation-administrations/[id]/progress/evaluation-progress.spec.ts:221]: use `if else`

**Review by jatnerubia — CHANGES_REQUESTED** [review]
^^

File: __tests__/pages/admin/evaluation-administrations/[id]/progress/evaluation-progress.spec.ts (hunk 2 above)
Comment (chadiegilnerubia) [reply __tests__/pages/admin/evaluation-administrations/[id]/progress/evaluation-progress.spec.ts:221]: Ohww yeap2 I forgot to use if else xD

Comment (chadiegilnerubia) [reply __tests__/pages/admin/evaluation-administrations/[id]/progress/evaluation-progress.spec.ts:221]: done :3

**Review by jatnerubia — APPROVED** [review]
Cool

**Pull request merged**